src/screens/Home/Home.js

```diff
@@ -37,78 +33,71 @@ export function Home({ navigation }) {
     return () => {
       unsubscribe();
     };
-  }, []);
+  }, [isError]);
 
-  const dateDiffInDays = (a, b) => {
-    const _MS_PER_DAY = 1000 * 60 * 60 * 24;
-    const utc1 = Date.UTC(a.getFullYear(), a.getMonth(), a.getDate());
-    const utc2 = Date.UTC(b.getFullYear(), b.getMonth(), b.getDate());
-    if (Math.floor((utc2 - utc1) / _MS_PER_DAY) < 2) {
-      return true;
-    } else {
-      return false;
-    }
-  };
+  const goDetails = useCallback(
+    (item) => {
+      navigation.navigate(NAVIGATION.details, { item: item });
+    },
+    [navigation]
+  );
+
+  const addMovieFavourites = useCallback(
+    (item) => {
+      if (moviesFav.includes(item, 0)) {
```

Comment:
I believe you don't need the `0` parameter because `includes` will always start from the beginning of the array.